components/ArticleListContainer.js

```diff
@@ -2,31 +2,19 @@ import styled from "styled-components";
 import ArticleCard from "./ArticleCard";
 import { useState } from "react";
 
-const initialArticles = [
-  {
-    id: "24343",
-    imageUrl: "/images/image-item1.jpg",
-    text: "rgasgsdfasdfdsfgasdfsadfagsdrtreterwtrewtfasdfasdfdsgasgadga",
-  },
-  {
-    id: "6454",
-    imageUrl: "/images/image-item2.jpg",
-    text: "nghsdgdfgdfagadsgfrtzt4etrewtertrewt4e64etzrehgsdrgdgdfsgdsfgdbfhrtehrhgrt",
-  },
-  {
-    id: "5466",
-    imageUrl: "/images/image-item4.jpg",
-    text: "bfdbadrgraegret4e6tr tztrze5rzterwterwterwteregradhraezrez",
-  },
-];
-
-export default function ArticleListContainer({ className }) {
-  const [articles, setArticles] = useState(initialArticles);
-
+export default function ArticleListContainer({
+  className,
+  articles,
+  showArticle,
+}) {
   return (
     <StyledListContainer className={className}>
```

Comment:
Why do you need the prop `className`? You do not handover a `className` value from your `index.js` to your `ArticleListContainer` and also your styled component `StyledListContainer` does not use this `prop`. I would recommend you to remove `className`.

Reply:
I dont need the prop className. But without it I can not style a react component with styled components. Dont know why, actually it should work. So I googled and found this solution.

Reply:
Hi yes I did this. But it has no effect which is really strange for me :(

Reply:
Hm. That's strange. Shall we take a look at it together after the lunch break?

Reply:
but I can remove it because now in dont need to restyle the component StyledListContainer somewhere else

Reply:
Ok! Then we leave it as it is for now. 👍